gears/src/baseapp/mod.rs

```diff
@@ -75,13 +73,6 @@ impl<DB: Database, PSK: ParamsSubspaceKey, H: ABCIHandler, AI: ApplicationInfo>
             .map(|e| e.max_gas)
             .unwrap_or_default();
 
-        block_height_set(multi_store.head_version());
```

Comment:
Are you sure about this change? Where block_heigh should be set?

Reply:
We no longer track the height since we can get it from the stored header or the request. This is inline with how Cosmos does it. We do now have a method for getting the last commit height of the store.